osmosis-1/osmosis.zone_config.json

```diff
@@ -190,6 +190,37 @@
         "name_suffix": "Oraichain Labs TON Bridge",
         "symbol_suffix": ".orai",
         "destination_network": "oraichain"
+      },
+      {
+        "provider": "Int3face",
+        "name_suffix": "Int3",
+        "symbol_suffix": ".int3",
+        "token": {
+          "chain_name": "int3face",
+          "base_denom": "uint3"
+        },
+        "destination_network": "int3face",
+        "integration": "Int3face Bridge",
+        "api_keys": {

```

Comment:
I'm not sure why API keys were added for this one. I'm thinking we can remove this.

Reply:
Removed